src/components/calculator.js

```diff
@@ -1,40 +1,54 @@
 import './calculator.css';
 import React from 'react';
+import calculate from '../logic/calculate';
+
 // eslint-disable-next-line react/prefer-stateless-function
 class Calculator extends React.Component {
+  constructor(props) {
+    super(props);
+    this.state = {};
+    this.handleClick = this.handleClick.bind(this);
+  }
+  // eslint-disable-next-line
```

Comment:
- [ ] Rather than disable the linters, strive to fix them as this makes you more accustomed to the style guide of the tech.

Reply:
@medaminedev66 which disable the one at line 5 or line 12?

Reply:
I can not remove line 5 because linter keeps asking me to use pure function instead of call component